cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -321,6 +321,9 @@ class EfsSettingsSchema(BaseSchema):
     deletion_policy = fields.Str(
         validate=validate.OneOf(DELETION_POLICIES), metadata={"update_policy": UpdatePolicy.SUPPORTED}
     )
+    accesspoint_id = fields.Str(
```

Comment:
The property would be exposed in the cluster config as `AccessPointId`
Please renamed it as `access_point_id`, here and in the other files.